### PR TITLE
Add AudioPlayer component

### PR DESCRIPTION
**Pull request by alshakerM**
Added AudioPlayer component.
Changed progressBar to slider from MUI

### CODE REVIEW

**Comment by alshakerM** [comment]
Fix sound lagging.
Added pause icon.
Styled audio timers

File: components/AudioPlayer/AudioPlayer.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        <Clear
+          className={styles.expandedClearIcon}
+          onClick={() => router.push('/lessons', undefined, { shallow: true })}
+        />
+      </div>
+      <div className={styles.expandedBody}>
+        <p className={styles.expandedSongName}>{openAudio.title}</p>
+        <p className={styles.expandedSonAuthor}>{openAudio.instructor}</p>
+      </div>
+      <div className={styles.playIconContainer}>
```
Comment (alshakero) [inline components/AudioPlayer/AudioPlayer.js:66]: This should be a button and the onClick should be on it. 99% of the time it's better for buttons to have onClicks for accessibilty.

File: components/AudioPlayer/AudioPlayer.module.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+.playIcon {
+  width: 100px !important;
+  height: 100px !important;
+  background-color: #32a4a7;
+  border-radius: 50%;
+}
+.playIcon.isPaused {
+  padding: 8px;
+}
```
Comment (alshakero) [inline components/AudioPlayer/AudioPlayer.module.css]: I think we should delete every trace of the word "expanded".

**Review by alshakero — CHANGES_REQUESTED** [review]
Pretty close to perfect. Left some comments.

**Comment by alshakerM** [comment]
Delete expanded from audio player.
Added button for onClick to play pause icons

File: components/AudioPlayer/AudioPlayer.js [hunk 3]
```diff
@@ -44,38 +44,38 @@ export function AudioPlayer({ openAudio }) {
...
-          <Pause
-            className={cx(styles.playIcon, { [styles.isPaused]: true })}
-            onClick={() => setPause(!pause)}
-          />
-        )}
+      <div
+        className={styles.playIconContainer}
+        onClick={() => setPause(!pause)}
+      >
+        <button className={styles.playButton}>
```
Comment (alshakero) [inline components/AudioPlayer/AudioPlayer.js]: The onClick should be on the button.

**Review by alshakero — APPROVED** [review]
LGTM. One note.

**Pull request merged**